wdl2cwl/main.py

```diff
@@ -236,16 +236,20 @@ def get_literal_value(expr: WDL.Expr.Base) -> Optional[Any]:
   if (contents == 'true') { return true;}
   if (contents == 'false') { return false;}
   throw "'read_boolean' received neither 'true' nor 'false': " + self[0].contents;
+}""",
+    "read_lines": r"""${
+  var contents = self[0].contents.replace(/\r\n$/, "").replace(/\n$/, "").replace(/\r$/, "");
+  // ^ remove any trailing newline to prevent a null being returned
+  return contents.split(/\r\n|\r|\n/);
 }""",
     "read_tsv": r"""${
-  var result;
-  self.contents.split(/\n|(\n\r)/).forEach(function(line) {
```

Comment:
the regex used capturing groups  `(  )` instead of the OR operator `|`. For split, the capturing groups are returned separately